src/crab/ebpf_domain.cpp

```diff
@@ -1416,13 +1979,14 @@ void ebpf_domain_t::do_load(const Mem& b, const Reg& target_reg) {
     });
 }
 
-template <typename A, typename X, typename Y>
-void ebpf_domain_t::do_store_stack(NumAbsDomain& inv, int width, const A& addr, X val_type, Y val_value,
+template <typename A, typename X, typename Y, typename Z>
+void ebpf_domain_t::do_store_stack(NumAbsDomain& inv, int width, const A& addr, X val_type, Y val_svalue, Z val_uvalue,
```

Comment:
The additional type parameter can be avoided if the caller that uses immediate first perform a cast to number_t. Up to you.

Reply:
(I didn't make a change here.)